plugins/che-plugin-analytics/che-plugin-analytics-wsagent/src/main/java/com/redhat/che/plugin/analytics/wsagent/AnalyticsEvent.java

```diff
@@ -19,7 +19,9 @@
   WORKSPACE_USED("Use Workspace in Che"),
   WORKSPACE_INACTIVE("Keep Workspace Inactive in Che"),
   WORKSPACE_STOPPED("Stop Workspace in Che"),
-  EDITOR_USED("Edit Workspace File In Che", 30, new String[] {PROGRAMMING_LANGUAGE});
+  EDITOR_USED("Edit Workspace File In Che", 30, new String[] {PROGRAMMING_LANGUAGE}),
+  PUSH_TO_REMOTE("Push to remote"),
```

Comment:
Isn't there some information from the endpoint call that we could use to add properties to the new event ?

Reply:
The only info is project path, which is IMO something we shouldn't pull as telemetry. We could attempt to resolve project type (e.g. maven) but I don't know how useful that is.